vignettes/manage.Rmd

```diff
@@ -0,0 +1,71 @@
+---
+title: "Manage lifecycle changes in functions you use"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Manage lifecycle changes in functions you use}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+options("lifecycle:::calling_package" = "tibble")
+```
+
+The lifecycle package uses warnings to tell you about deprecated functions.
+Deprecated functions will be removed in a future release, so it's good practice to eliminate the warnings as soon as you see them.
+
+For example, lets imagine your code uses `tibble::data_frame()`, which was deprecated in favour of `tibble()` in version 1.1.0.
+`data_frame()` now looks something like this:
+
+```{r}
+data_frame <- function(...) {
+  lifecycle::deprecate_warn("1.1.0", "data_frame()", "tibble()")
+  tibble::tibble(...)
+}
+```
+
+That means if you use `data_frame()` in your own code you'll get a warning:
+
+```{r}
+df1 <- data_frame(x = 1, y = 2)
+df2 <- data_frame(a = "apple", b = "banana")
```

Comment:
I removed some hard-wired output comments here that replicated the live warning being thrown here.

Reply:
For complicated reasons, the footer (i.e. the bit that says "this warning is displayed..." is not displayed in RMarkdown files)